docs/en/Configuring-Alluxio-with-OBS.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: global
+title: Configuring Alluxio with OBS
+nickname: Alluxio with OBS
+group: Under Store
+priority: 5
+---
+
+* Table of Contents
+{:toc}
+
+This guide describes how to configure Alluxio with
+[Huawei OBS](http://www.huaweicloud.com/en-us/product/obs.html) as the under storage system. Object Storage
+Service (OBS) is a massive, secure and highly reliable cloud storage service provided by Huawei Cloud.
+
+## Initial Setup
+
+To run an Alluxio cluster on a set of machines, you must deploy Alluxio binaries to each of these
+machines.You can either
+[compile the binaries from Alluxio source code](http://alluxio.org/documentation/master/Building-Alluxio-Master-Branch.html),
+or [download the precompiled binaries directly](http://alluxio.org/documentation/master/Running-Alluxio-Locally.html).
+
+Also, in preparation for using OBS with alluxio, create a bucket or use an existing bucket. You
+should also note that the directory you want to use in that bucket, either by creating a new
+directory in the bucket, or using an existing one. For the purposes of this guide, the OBS bucket
+name is called `OBS_BUCKET`, and the directory in that bucket is called `OBS_DIRECTORY`. Also, for
```

Comment:
Maybe name these variables with brackets, like `<OBS_BUCKET>`, `<OBS_DIRECTORY>`, and `<OBS_ENDPOINT>` ?